react/features/feedback/functions.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import jwtDecode from 'jwt-decode';
+
+import logger from './logger';
+
+/**
+ * Sends feedback metadata to JaaS endpoints.
+ *
+ * @param {string} url - The JaaS metadata endpoint URL.
+ * @param {Object} feedbackData - The feedback data object.
+ * @returns {Promise}
+ */
+export async function sendFeedbackToJaaSRequest(url: string, feedbackData: Object) {
+    const {
+        jwt,
+        sessionId,
+        meetingFqn,
+        score,
+        message
+    } = feedbackData;
+    const headers = {
+        'Authorization': `Bearer ${jwt}`,
+        'Content-Type': 'application/json'
+    };
+    const payload = jwtDecode(jwt);
```

Comment:
Again, isn't the jwt already decoded and stored in redux?

Reply:
yep, apparently tenant was not , but fixed it by checking context.tenant || sub || undefined.